webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux/f1';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo/f2';
+
+// import from a file named `bar.js` located in the parent directory:
+import f3 from '../bar/f3';
+
+// import from a file named `index.js` located in a directory named `baz` that
+// is in the current directory:
+import f4 from './baz/index/f4';
```

Comment:
Drop the `/index/f4`, that's just what we're using for our variable, that's not what we're actually importing.

We don't need to include `/index`, because Webpack by default will look for an `index.js` file, and use that as our export.